bin/common.sh

```diff
@@ -146,7 +175,7 @@ find_module_jar() {
     BUILT_JAR=$(find_module_jar_at ${BK_HOME}/${MODULE_PATH}/target ${MODULE_NAME})
     if [ -z "${BUILT_JAR}" ]; then
       echo "Couldn't find module '${MODULE_NAME}' jar." >&2
-      read -p "Do you want me to run \`mvn package -DskiptTests\` for you ? " answer
+      read -p "Do you want me to run \`mvn package -DskiptTests\` for you ? (y|n) " answer
       case "${answer:0:1}" in
         y|Y )
           mvn package -DskipTests -Dstream
```

Comment:
it would print out the result in console for this case. it will not looks like the script has hung.

you probably refer to https://github.com/apache/bookkeeper/blob/master/bin/common.sh#L186

Reply:
it doesn't though. This method is called from $(find_module_release_jar), so stdout is captured into the RELEASE_JAR variable, and only when the call completes.